### PR TITLE
Obsolescence Notification in Readme

### PR DESCRIPTION
**Pull request by CommanderStorm**
### Issue
See https://github.com/TUM-Dev/Campus-Android/pull/1599

Added a PSA about the planned replacement and the current state of development to the README.md.
This is partially preparing for the OSLab next semester, since many students want to contribute to TC Apps.

This is just a rough draft (if you would like something to be worded differently/…, please add a comment)
Likely, this draft should be toned down.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,18 @@ The TUM Campus App (TCA) is an open source project, developed by volunteers and
 
 It mostly targets phones, but can also be used on tablets or any other device that runs iOS. We also have a [mac app](https://github.com/TUM-Dev/Campus-iOS/releases/lastest). This is the repo for the iOS version of the TUM Campus App.
 
+
+## 🚨 Public Service Announcement 🚨
```
Comment (philippzagar) [inline README.md:16]: Suggestion:

Currently, we are at a transitioning point for our TUM Campus Apps. We have three platform-specific versions: [iOS](https://github.com/TUM-Dev/Campus-iOS), [Android](https://github.com/TUM-Dev/Campus-Android), and [UWP](https://github.com/TUM-Dev/Campus-UWP). However, maintaining these distinct versions has proven challenging, with notable issues including feature disparity and the need for individual bug fixes for each platform. Furthermore, this model is dependent on having a dedicated maintainer for each app.

As such, we have decided to consolidate our resources and efforts towards the creation of a unified, cross-platform app using [Flutter](https://flutter.dev/). You can access the new codebase here: [github.com/TUM-Dev/Campus-Flutter](https://github.com/TUM-Dev/Campus-Flutter).

We invite and value your contributions to the ongoing development of the Campus-iOS. However, please note that we are transitioning towards a new Flutter-based codebase, expected to take over the platform-specific apps in 2024. 

By uniting our efforts under one platform, we aim to enhance our efficiency and streamline the feature sets across all versions, thereby offering a more cohesive and reliable user experience. We thank you for your understanding and look forward to your continued support during this exciting period of transition.

**Review by philippzagar — COMMENTED** [review]
Had some suggestions regarding the formulations

File: README.md (hunk 1 above)
Comment (philippzagar) [reply README.md:16]: Let's try to use one consistent text for all READMEs of the native TCAs :)

File: README.md [hunk 2]
```diff
@@ -12,6 +12,18 @@ The TUM Campus App (TCA) is an open source project, developed by volunteers and
...
+
+The current state of the game is, that we have three Campus apps ([iOS](https://github.com/TUM-Dev/Campus-iOS), [Android](https://github.com/TUM-Dev/Campus-Android), [UWP](https://github.com/TUM-Dev/Campus-iOS)).
+This current model brings a few downsides, but most notorios are feature divergence and bugfixes.
+It also relies on there always being a core maintainer to each of the projects.
+
+Feature divergance and the lack of a maintainer on the Android side have pushed us towards uniting our front and merging the effort into a [Flutter](https://flutter.dev/)-based alternative.
+
+You are still welcome to contribute to this Campus-iOS.  
+This new codebase will take over these apps sometime in 2024: [github.com/TUM-Dev/Campus-Flutter](https://github.com/TUM-Dev/Campus-Flutter)
```
Comment (CommanderStorm) [inline README.md:25]:
```suggestion
Currently, we are at a transitioning point for our TUM Campus Apps. We have three platform-specific versions: [iOS](https://github.com/TUM-Dev/Campus-iOS), [Android](https://github.com/TUM-Dev/Campus-Android), and [UWP](https://github.com/TUM-Dev/Campus-UWP).  
 However, maintaining these distinct versions has proven challenging, with notable issues including feature disparity and the need for individual bug fixes for each platform. Furthermore, this model is dependent on having a dedicated maintainer for each app.

As such, we have decided to consolidate our resources and efforts towards the creation of a unified, cross-platform app using [Flutter](https://flutter.dev/). You can access the new codebase here: [github.com/TUM-Dev/Campus-Flutter](https://github.com/TUM-Dev/Campus-Flutter).

By uniting our efforts under one platform, we aim to enhance our efficiency and streamline the feature sets across all versions, thereby offering a more cohesive and reliable user experience.

We invite and value your contributions to the ongoing development of the Campus-iOS.
However, please note that we are transitioning towards a new [Flutter](https://flutter.dev/)-based codebase, expected to take over the platform-specific apps in 2024.
We thank you for your understanding and look forward to your continued support during this exciting period of transition.
If you have any comments/feedback about this decison, you can reach us at app <at> tum.de or by adding a GitHub issue
```

**Comment by CommanderStorm** [comment]
I have also added a line for contacting at then end.

**Review by philippzagar — APPROVED** [review]
LGTM!

**Review by tgymnich — APPROVED** [review]
giphy

**Comment by CommanderStorm** [comment]
@TUM-Dev/campus-app 
I would suggest that if nobody cries out in the next ~week, I will roll out the current formulation across
- https://github.com/TUM-Dev/Campus-Android
- https://github.com/TUM-Dev/Campus-UWP
- https://github.com/TUM-Dev/Campus-iOS

**Review by joschahenningsen — APPROVED** [review]

**Comment by CommanderStorm** [comment]
Okay.. so I guess no waiting for a week ^^

**Pull request merged**